### PR TITLE
[DOCU-912] Kafka Log plugin remove install section

### PR DESCRIPTION
**Pull request by tlblessing**
https://konghq.atlassian.net/browse/DOCU-912 Remove unnecessary install section (already removed from Kafka Upstream plugin). Also removed dupe global topic.

The more I look at this doc, the more room for improvements I see. Will log follow-up ticket. https://konghq.atlassian.net/browse/DOCU-1483

Direct review link:

https://deploy-preview-2848--kongdocs.netlify.app/hub/kong-inc/kafka-log/

### CODE REVIEW

File: app/_hub/kong-inc/kafka-log/index.md [hunk 1]
```diff
@@ -209,25 +140,23 @@ The following guidelines assume that both `Kong` and `Kafka` have been installed
...
 
-3. Add the `kafka-log` plugin globally:
+2. Add the `kafka-log` plugin globally:
 
     ```
     curl -X POST http://localhost:8001/plugins \
         --data "name=kafka-log" \
-        --data "config.bootstrap_servers[1].host=localhost \
-        --data "config.bootstrap_servers[1].port=9092 \
+        --data "config.bootstrap_servers[1].host=localhost" \
```
Comment (tlblessing) [inline app/_hub/kong-inc/kafka-log/index.md:148]: fixed syntax for missing quotes

File: app/_hub/kong-inc/kafka-log/index.md [hunk 2]
```diff
@@ -238,5 +167,19 @@ The following guidelines assume that both `Kong` and `Kafka` have been installed
         --timeout-ms 1000
     ```
 
+## Log format
+
+Similar to the [HTTP Log Plugin](https://docs.konghq.com/hub/kong-inc/http-log#log-format).
```
Comment (tlblessing) [inline app/_hub/kong-inc/kafka-log/index.md:172]: need to create a section just for this plugin, not link off to another.

File: app/_hub/kong-inc/kafka-log/index.md [hunk 3]
```diff
@@ -238,5 +167,19 @@ The following guidelines assume that both `Kong` and `Kafka` have been installed
...
+
+This plugin makes use of the [lua-resty-kafka](https://github.com/doujiang24/lua-resty-kafka) client under the hood.
+
+## Known issues and limitations
+
+1. There is no support for TLS.
+2. There is no support for authentication.
+3. There is no support for message compression.
+
 [badge-travis-url]: https://travis-ci.com/Kong/kong-plugin-kafka-log/branches
```
Comment (tlblessing) [inline app/_hub/kong-inc/kafka-log/index.md:184]: not sure we need these links. these comments will be addressed in a subsequent ticket.

File: app/_hub/kong-inc/kafka-log/index.md [hunk 4]
```diff
@@ -124,82 +125,12 @@ params:
...
-```
-kong reload
-```
-
-## Configuration
-
-### Enabling globally
-
-```bash
-$ curl -X POST http://kong:8001/plugins \
```
Comment (tlblessing) [inline app/_hub/kong-inc/kafka-log/index.md:162]: this is what the autogen example should look like, not have all cap entries in the value in examples parameters fields.

File: app/_hub/kong-inc/kafka-log/index.md [hunk 5]
```diff
@@ -177,9 +177,12 @@ This plugin makes use of the [lua-resty-kafka](https://github.com/doujiang24/lua
 
 ## Known issues and limitations
 
-1. There is no support for TLS.
-2. There is no support for authentication.
-3. There is no support for message compression.
+There is currently no support for:
+
```
Comment (tlblessing) [inline app/_hub/kong-inc/kafka-log/index.md:181]: changed to ul and removed redundant test

File: app/_hub/kong-inc/kafka-log/index.md [hunk 6]
```diff
@@ -39,7 +39,8 @@ params:
       default:
       datatype: set of record elements
       description: |
-        Set of bootstrap brokers in a `{host: host, port: port}` list format. For examples, see the Quickstart section.
+        Set of bootstrap brokers in a `{host: host, port: port}` list format. For examples, see the
+        [Quickstart](#quickstart).
```
Comment (tlblessing) [inline app/_hub/kong-inc/kafka-log/index.md:43]: made a link here

**Review by murillopaula — APPROVED** [review]

**Review by HCloward — APPROVED** [review]

**Pull request merged**